projects.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: page
+title: Current Projects
+descriptiveTitle: Toronto Mesh Projects
+order: 1
+parent: Get Involved
+---
+
```

Comment:
Consider a TOC given the size of the list?

Reply:
I like it- added!